.github/workflows/cypress-run.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         project: ${{ fromJSON(inputs.projects) }}
     name: ${{ matrix.project }}
-    runs-on: self-hosted-runner
+    runs-on: mainnet-compatible-runner
```

Comment:
We need to update the workflow so PRs on Develop are still on the self-hosted runner

Reply:
they will, as this change is going to main only